builder/parser/line_parsers.go

```diff
@@ -209,34 +208,28 @@ func parseString(rest string) (*Node, map[string]bool, error) {
 
 // parseJSON converts JSON arrays to an AST.
 func parseJSON(rest string) (*Node, map[string]bool, error) {
-	var (
-		myJson   []interface{}
-		next     = &Node{}
-		orignext = next
-		prevnode = next
-	)
-
+	var myJson []interface{}
```

Comment:
Generally though, `RUN [ some-test-here ]` _will_ work, as long as it's not valid JSON syntax.